style.css

```diff
@@ -1,9 +1,76 @@
+body {
+    width: 1440px;
+}
 
-.header{
-    display:flex;
-    flex-direction: row;
+.header {
+    display: flex;
+    height: 30px;
+    width: 1440px;
 }
-.nav{
-    display:flex;
+
+.nav {
+    display: flex;
     flex-direction: row;
+
+    justify-content: space-between;
+    width: 100%;
+
+}
+
+.link_flex_container {
+    display: flex;
+    justify-content: flex-end;
+    width: 80%;
+
+}
+.logo{
+    font-size: 25px;
+}
+.links {
+    margin: 0px 10px 0px 10px;
```

Comment:
when you set zero value in css don't put the value type, so
`margin: 0 10px 0 10px;`, more than that, it's repetitive so we can shorten it some more:
`margin: 0 10px;` _third value will be same as first, fourth is the same as second_

Reply:
didn't know :) fixed.